packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -156,6 +156,11 @@ export class ArtifactCreator {
             (mismatchingAverageDistanceRatio * 100).toFixed(2)}%`);
       }
 
+      const {rmsDistanceRatio} = thresholdResults[0];
+      console.log(
+          `\n  📊 Decibels of root mean square color distance (without threshold): ${
+              (10 * Math.log10(rmsDistanceRatio * 100)).toFixed(2)}`);
```

Comment:
The original db equation is 10 * Math.log10(I * 1000000000000). And here is the output using this equation:
![Screenshot 2020-06-09 at 3 45 32 PM](https://user-images.githubusercontent.com/44676379/84193404-6621ca80-aa69-11ea-812d-921f3c5b0ee1.png)
I want to modify this equation for two reasons: 
1. the unit of the original equation is different, so I don't think it's meaningful to stick to the same number
2. Even if the rmsDistanceRatio is very small, the decibels can be very big. so the result may be less "interesting" as you said

So I modified it to 10 * Math.log10(rmsDistanceRatio * 100), and here is the result of the same scenario:
![Screenshot 2020-06-09 at 3 47 30 PM](https://user-images.githubusercontent.com/44676379/84193710-dfb9b880-aa69-11ea-9a30-b3406f7e8520.png)
So do you agree with the equation i use here?